totalRP3/modules/register/characters/register_characteristics.lua

```diff
@@ -1523,6 +1548,8 @@ function TRP3_API.register.inits.characteristicsInit()
 	TRP3_RegisterCharact_Edit_ResidenceFieldText:SetText(loc.REG_PLAYER_RESIDENCE);
 	TRP3_RegisterCharact_Edit_BirthplaceFieldText:SetText(loc.REG_PLAYER_BIRTHPLACE);
 
+	setupListBox(TRP3_RegisterCharact_Dropdown_RelationshipField, RELATIONSHIP_STATUS_DROPDOWN, onRelationshipStatusSelection, loc.REG_PLAYER_RELATIONSHIP_STATUS_UNKNOWN, 200, false);
+	TRP3_RegisterCharact_Dropdown_RelationshipFieldTitle:SetText(loc.REG_PLAYER_RELATIONSHIP_STATUS);
```

Comment:
It's minor, but the template you have implemented in XML has some code for displaying a tooltip. If you have any idea of a tooltip text to explain to the user how to use this field you could add it here (either using the old TRP3 way or Ellyb's Tooltips module).

You could also go for a description of each of the relationship statuses by passing a third field to your `RELATIONSHIP_STATUS_DROPDOWN` table above that would be displayed in a tooltip when the user goes to select an option, if you feel fancy.

Reply:
I don't feel fancy.